packages/hub/src/scope.ts

```diff
@@ -185,9 +185,14 @@ export class Scope implements ScopeInterface {
   /**
    * @inheritDoc
    */
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  public setContext(key: string, context: { [key: string]: any } | null): this {
-    this._contexts = { ...this._contexts, [key]: context };
+  public setContext(key: string, context: Context | null): this {
+    if (context === null) {
+      // eslint-disable-next-line @typescript-eslint/no-dynamic-delete
+      delete this._contexts[key];
```

Comment:
Hmm this is more than changing the types, would the previous code delete keys?! Was this a bug?

Reply:
From https://develop.sentry.dev/sdk/unified-api/
![image](https://user-images.githubusercontent.com/88819/94807033-4a16b000-03ef-11eb-9fa1-894d5de91f37.png)

So now setting a context to null deletes it. Let's document this in the changelog.

Reply:
Was a bug, fixed.